config/releases.exs

```diff
@@ -83,7 +83,7 @@ if System.get_env("DISABLE_MQTT") != "true" do
     password: System.get_env("MQTT_PASSWORD"),
     tls: System.get_env("MQTT_TLS"),
     accept_invalid_certs: System.get_env("MQTT_TLS_ACCEPT_INVALID_CERTS"),
-    namespace: System.get_env("MQTT_NAMESPACE") |> Util.validate_namespace!()
+    namespace: System.get_env("MQTT_NAMESPACE", "teslamate") |> Util.validate_namespace!()
```

Comment:
`validate_namespace/1` raises if `MQTT_NAMESPACE`contains a `/` which would make it impossible to use e.g. `teslamate/account_0`.

Reply:
My bad, didn't catch that - was there a reason for adding this validation? I will remove in this PR for now.